test_scripts/API/Restructuring_OnResetTimeout/022_BC_DialNumber_without_BC_OnResetTimeout.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) DialNumber is requested
+-- 2) Some time after receiving DialNumber request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 15000) to SDL
+-- 4) HMI sends response
+-- SDL does:
+-- 1) SDL must always wait for HMI response to BC.DialNumber as long as it takes and not return GENERIC_ERROR to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local paramsForRespFunction = {
+	respTime = 13000,
+	notificationTime = 0,
+	resetPeriod = 12000
+}
+
+local function DialNumber( pRespExpect, pRespTime, pResponseFunc, pParams, pRespParams )
+  if not pRespParams then pRespParams = { success = true, resultCode = "SUCCESS" } end
+  local cid = common.getMobileSession():SendRPC("DialNumber", { number = "#3804567654*" })
+  EXPECT_HMICALL("BasicCommunication.DialNumber", { appID = c.getHMIAppId(), number = "#3804567654*" })
```

Comment:
@HSavynetska if you will use this function it will not work because of `appID = c.getHMIAppId()`,
 should be `appID = common.getHMIAppId()`

Reply:
@LitvinenkoIra Update